validator/client/validator_propose.go

```diff
@@ -77,6 +77,7 @@ func (v *validator) ProposeBlock(ctx context.Context, slot uint64, pubKey [48]by
 		"blockRoot":       blkRoot,
 		"numAttestations": len(b.Body.Attestations),
 		"numDeposits":     len(b.Body.Deposits),
+		"signature":       fmt.Sprintf("%#x", bytesutil.Trunc(b.Signature)),
```

Comment:
How does a truncated signature help?

Reply:
Use-case i was looking to is to match signature failure logs on beacon node with the proposing log/attesting log on the validator client

Reply:
I'd rather have the full signature as a debug log @terencechain what do you think?

Reply:
That's fine too. Done!